app/app.go

```diff
@@ -428,7 +428,7 @@ func NewLinkApp(
 	ibcRouter := porttypes.NewRouter()
 	ibcRouter.AddRoute(icahosttypes.SubModuleName, icaHostIBCModule).
 		AddRoute(ibctransfertypes.ModuleName, transferIBCModule).
-		AddRoute(wasmplustypes.ModuleName, wasm.NewIBCHandler(app.WasmKeeper, app.IBCKeeper.ChannelKeeper))
+		AddRoute(wasmplustypes.ModuleName, wasm.NewIBCHandler(app.WasmKeeper, app.IBCKeeper.ChannelKeeper, app.IBCKeeper.ChannelKeeper))
```

Comment:
I just wonder, is it okay to pass `app.IBCKeeper.ChannelKeeper` as the third parameter where `appVersionGetter` is expected. `ChannelKeeper` does implements `appVersionGetter` interface though. Just curious.

Reply:
Please check [appVersionGetter interface](https://github.com/Finschia/wasmd/blob/dedcd9ec5ba017f8a3591534ec099bc9097adb27/x/wasm/ibc.go#L21-L24) and [ChannelKeeper interface](https://github.com/Finschia/ibc-go/blob/7176e4ec5b162fe5f1a335c56e4b7b201c202d1f/modules/core/04-channel/keeper/keeper.go#L90).